package/autoyast2.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jun 26 11:21:11 UTC 2020 - Josef Reidinger <jreidinger@suse.com>
+
+- Does not depend cloning on SetModified API call (bsc#1172552)
```

Comment:
I would rephrase it the other way around: "Cloning does not depend on the SetModified API call"